### ISSUE TITLE
Wrong links on project pages

### ISSUE THREAD

**Issue by gbehrmann** [argus-authz/argus-pep-api-java#1]
Hi,

On the project page http://argus-authz.github.io/argus-pep-api-java/ there is a link to a Maven repository at http://www.switch.ch/grid/authz/docs/maven2/release. I get 404 replies from this repository.

I thus tried to contact support at argus-support@cern.ch, but got a 

> argus-support wasn't found at cern.ch.

reply back from Office 365.

What is the official Maven repository for Argus client artifacts, and could you please update the project pages with the correct links?

Thanks,

Gerd

**Comment by valerytschopp** [comment]
Hi gberhermann,

The support email address have change to argus-support@googlegroups.com some years ago, but we forgot to update this page. It is done now.

As for the maven repository, I've to ask my colleague about it.

Cheers,
Valery

**Comment by gbehrmann** [comment]
Thanks for the quick reply, @valerytschopp. Let me know if you find a Maven repo somewhere.

**Comment by msalle** [comment]
Hi Gerd,
I actually think you might not need any extra maven repo apart from the central repo. I've managed to build quite a few of the packages using only http://uk.maven.org/maven2, as specified in the argus-parent pom. That pom still contains the Heras repo which also needs to be disabled.
Best wishes,
Mischa

**Comment by valerytschopp** [comment]
Hi Gerd,

You still can build it locally, without the SWITCH maven repo, which doesn't exist anymore:

```
git clone https://github.com/argus-authz/argus-parent.git
cd argus-parent
mvn install

cd ..
git clone https://github.com/argus-authz/argus-pep-common.git
cd argus-pep-common
mvn install

cd ..
git clone https://github.com/argus-authz/argus-pep-api-java.git
cd argus-pep-api-java
mvn compile
```

**Comment by gbehrmann** [comment]
@valerytschopp, yeah, that's what we did in the past. I will make a custom build and push it to our own repository at dcache.org (we got a third party repo for libs not available in pre-build form). I just wanted to double check whether it was available in a Maven repository somewhere as that would surely have been easier.

Thanks,

/gerd

**Comment by gbehrmann** [comment]
BTW: Remember to update the project page to get rid of the reference to the SWITCH repo :-)

**Comment by valerytschopp** [comment]
Done

**Comment by gbehrmann** [comment]
Let me reopen this issue for a moment. The whole reason I am looking into this is because we want to move to CANL 2 and bouncycastle 1.50. For this I need pep-common 2.3.1 from the 2_3 branch and pep-java 2.3.0 also from the 2_3 branch. Both depend on 1.7-SNAPSHOT of argus-parent. Depending on a snapshot version prevents upload to our Nexus release repositories. Could you make a release of the parent and update pep-common and pep-java?

I could of course just make a local adjustment, but I would hate having to fork your code base just to adjust some version numbers.

Thanks,

Gerd

**Comment by andreaceccanti** [comment]
Hi Gerd,

I'll let you know when the 1.7 release parent release is ready.
